src/legacy/status_im/fleet/core.cljs

```diff
@@ -91,5 +91,4 @@
        cofx
        :fleet
        fleet
-       {:on-success
-        #(re-frame/dispatch [:logout])}))))
+       {:on-success #(re-frame/dispatch [:profile.logout/logout])}))))
```

Comment:
This is part of the same rename, the previous event in `legacy` had two names, but `:logout` is a simple keyword hard to track, this is why it hasn't been used